ES/ESkae011.xml

```diff
@@ -272,6 +270,58 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                            leaves) similarly build a separate unit, the result of a separate
                            production process.</decoNote>
                      </decoDesc>
+
+                     <additions>
+                        <list>
+
+                           <item xml:id="a1">
+                              <locus from="1v" to="2v" facs="002"/>
+                              <desc type="Letter">A letter of <persName ref="PRS9451Tewoflos">
+                                    <roleName type="office">ʾǝč̣č̣age</roleName> Tewoflos
+                              </persName>concerning the “Orthodox faith”, in Amharic, written probably in his own hand
+                              some time between <date>1872</date> and his deposition in <date>1889</date>. The letter is
+                              addressed to the churches of <placeName ref="LOC5882Tamben"/>
+                            contans a summary of the arguments of <foreign xml:lang="am">hulatt lədat</foreign> doctrine
+                             (<foreign xml:lang="am">kārrā</foreign>). It mentions
+                          <persName ref="PRS13144GabraWald">Gabra Wald</persName>,
+                          obviously the owner of the manuscript, as the person who was supposed to
+                        fetch the missive in the book.</desc>
+                                 <q xml:lang="am"><seg part="I">ሃይማኖት፡ ማለት፡ አዋን፡ ብሎ፡ የሚቀበሉት፡ ኡነት፡ ብለው፡ የሚፀኑበት፡ ማለት᎓
+                                    ነው፡ ጦማረ፡ ሃይማኖት፡ ዘፈነዋ፡ እጨጌ፡ ቴዎፍሎስ፡ ዘነበረ፡ በመንበረ፡ ተክለ፡ ሃይማኖት፡ ውስተ፡ ኰሎን፡ አብያተ፡ ክርስቲያናት፡
+                                  ኀገረ፡ ተንቤን። በእዴሁ፡ ለአቡነ፡ መምሕር፡ ገብረ፡ ወልድ፡ አጽምኡ፡ ሕዝብየ፡ <gap reason="omitted" resp="DN"></gap></seg></q>
+
+                           </item>
+
+                           <item xml:id="a2">
+                              <locus target="#59ra" facs="059"/>
+                              <desc type="DonationNote">The name of the donor, <persName ref="PRS9451Tewoflos">Gabra Wald</persName>, is
```

Comment:
I think it's just the wrong ID for Gabra Wald, can you check throughout the manuscript as well?

Reply:
corrected